src/compiler/checker.ts

```diff
@@ -11707,7 +11707,7 @@ module ts {
                 return getEnumMemberValue(<EnumMember>node);
             }
 
-            let symbol = getNodeLinks(node).resolvedSymbol;
+            let symbol = getSymbolAtLocation(node);

```

Comment:
ok, I guess I'm wrong since node here will be `PropertyAccessExpression` / `ElementAccessExpression` not `Identifier`